gitlab/docs/runner/counter.gitlab_runner_request_concurrency_exceeded_total.md

```diff
@@ -0,0 +1,8 @@
+---
+title: gitlab_runner_request_concurrency_exceeded_total
+brief: Counter tracking exceeding of request concurrency.
```

Comment:
Another general comment - for consistency, brief descriptions shouldn't have a period at the end.